menas/ui/components/dataset/datasetInfo.fragment.xml

```diff
@@ -14,41 +14,80 @@
+              <Text id="currentDatasetDescription" text="{dataset>/description}"/>
+            </f:fields>
+          </f:FormElement>
+          <f:FormElement label="Version">
+            <f:fields>
+              <Text id="currentDatasetVersion" text="{dataset>/version}" />
+            </f:fields>
+          </f:FormElement>
+          <f:FormElement label="HDFS raw data folder path">
+            <f:fields>
+              <Text id="currentDatasetRawPath" text="{dataset>/hdfsPath}" />
+            </f:fields>
+          </f:FormElement>
+          <f:FormElement label="HDFS conformed data publish folder path">
+            <f:fields>
+              <Text id="currentDatasetPublishedPath" text="{dataset>/hdfsPublishPath}" />
+            </f:fields>
+          </f:FormElement>
+          <f:FormElement label="Schema">
+            <f:fields>
+              <Link text="{dataset>/schemaName} (v{dataset>/schemaVersion})"
                 press="toSchema"
                 cust:name="{dataset>/schemaName}"
                 cust:version="{dataset>/schemaVersion}"
                 id="currentDatasetSchema"/>
-            <Label text="Last Update"/>
-            <Text id="currentDatasetLastUpdate"
-                  text="{path: 'dataset>/lastUpdated', formatter: 'Formatters.stringDateShortFormatter'}"/>
-            <Label text="Last Update By"/>
-            <Text id="currentDatasetUserUpdated"
-                  text="{dataset>/userUpdated}"/>
-            <Label text="Created"/>
-            <Text id="currentDatasetCreated"
-                  text="{path: 'dataset>/dateCreated', formatter: 'Formatters.stringDateShortFormatter'}"/>
-            <Label text="Created By"/>
-            <Text id="currentDatasetUserCreated"
-                  text="{dataset>/userCreated}"/>
-        </form:content>
-    </form:SimpleForm>
+            </f:fields>
+          </f:FormElement>
+          <f:FormElement label="Last Updated">
+            <f:fields>
+              <Text id="currentDatasetLastUpdate" text="{path: 'dataset>/lastUpdated', formatter: 'Formatters.stringDateShortFormatter'}" />
+            </f:fields>
+          </f:FormElement>
+          <f:FormElement label="Last Updated By">
+            <f:fields>
+              <Text id="currentDatasetUserUpdated" text="{dataset>/userUpdated}" />
+            </f:fields>
+          </f:FormElement>
+          <f:FormElement label="Created">
+            <f:fields>
+              <Text id="currentDatasetCreated" text="{path: 'dataset>/dateCreated', formatter: 'Formatters.stringDateShortFormatter'}" />
+            </f:fields>
+          </f:FormElement>
+          <f:FormElement label="Created By">
+            <f:fields>
+              <Text id="currentDatasetUserCreated" text="{dataset>/userCreated}" />
+            </f:fields>
+          </f:FormElement>
+        </f:formElements>
+      </f:FormContainer>
+      <f:FormContainer formElements="{/currentDataset/propertiesDisplay}">
```

Comment:
What are the chances to have them in the same order as when editing dataset?

Reply:
Will check, this will probably create those margins, but we should be able to force-style otherwise.

![image](https://user-images.githubusercontent.com/2735155/107223945-1855cb80-6a17-11eb-89e1-d267ee997e5d.png)

Reply:
Ok, restyled those margins to be less dramatic, tried without them but I think they actually make it easier to read
![image](https://user-images.githubusercontent.com/2735155/107230296-2e678a00-6a1f-11eb-9398-20a1b776112d.png)

Reply:
While the change looks good to me. What I talked about is sorting the list of propertiesDisplay the same way when you edit dataset

#### Detail
![image](https://user-images.githubusercontent.com/17467768/107352228-43eabb80-6ad4-11eb-82c5-279920a34287.png)

#### Dataset Edit
![image](https://user-images.githubusercontent.com/17467768/107352395-7399c380-6ad4-11eb-9d2b-274137dcefd7.png)

Reply:
Will have a look

Reply:
Ok added a sorter. This will only work on a best-effort basis. In cases where the dataset loads faster than the property definitions, a default order will be used.